src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { funFacts,  FunFact  as FunFactInterface } from "./funfacts";
```

Comment:
A few spaces too many here!
```suggestion
import { funFacts, FunFact as FunFactInterface } from "./funfacts";
```

Also, the file should use `camelCase`:
```suggestion
import { funFacts,  FunFact  as FunFactInterface } from "./funFacts";
```
Remember to rename the actual file, and anywhere else it's imported!

Reply:
In fact, you shouldn't be importing `funFacts` at all, since below in the file it's passed in as a prop!
```suggestion
import { FunFact  as FunFactInterface } from "./funfacts";
```

Reply:
done, thanks